docs/docs/pyproject.md

```diff
@@ -192,6 +192,15 @@ poetry = 'poetry:console.run'
 
 Here, we will have the `poetry` script installed which will execute `console.run` in the `poetry` package.
 
+To specify a script that
+[depends on an extra](#extras),
+you may provide an entry as a dict:
```

Comment:
these line breaks are not necessary and visually irritating.

Reply:
Some would argue that short, broken lines are easier to read and maintain 🙂 
Smaller diffs, and sentences are well cut on meaningful parts or punctuation.
It can also help in avoiding unnecessary words
that make complicated sentences
and therefore bad documentation!